internal/modules/cpu.go

```diff
@@ -48,11 +48,7 @@ func cpu(mo *ModuleOutput, cfg ModuleConfig) {
 	cache.Add(cpuTimeKey, newCpuTime, "1h")
 	//Generate output
 	mo.Color = getColor(cpu, cfg)
-	if cfg.ShortFormat {
-		mo.FullText = fmt.Sprintf("%s", mo.FullText)
-	} else {
-		mo.FullText = fmt.Sprintf("%s%.2f%%", mo.FullText, cpu)
-	}
+	mo.FullText = fmt.Sprintf("%s%.2f%%", mo.FullText, cpu)
```

Comment:
if you want short form as only icon with color no need to change module code at all